packages/core/test/actions.test.ts

```diff
@@ -223,6 +223,46 @@ describe('entry/exit actions', () => {
     }
   });
 
+  const deepMachine2 = createMachine({
```

Comment:
Could you use a single machine per test? I know that a lot of tests are written in this way here but we are trying to not add any new ones like this.

The preferred approach to write tests right now looks something like this: https://github.com/statelyai/xstate/pull/3424/files#diff-0f3376f69e4a3b17ec5757d5e7efd1d4438b4623627438bd1de139674b975ccbR386

Reply:
Yeah, no worries. I'll update it.

Reply:
Done.